stix_shifter_modules/sentinelone/stix_transmission/results_connector.py

```diff
@@ -0,0 +1,197 @@
+        Format the results in json format
+        :param item: list of dictionary items
+        :return: list
+        """
+        for item in res_dict['data']:
+            if item.get("netProtocolName") is None:
+                if item.get("srcPort") is not None or item.get("dstPort") is not None:
+                    item["netProtocolName"] = "null"
+
+            if item.get("registryKeyPath") is not None:
+                registryKeyPath = item.get("registryKeyPath")
+                registryKeyPath = registryKeyPath.replace('MACHINE\\', 'HKEY_LOCAL_MACHINE\\')
+                registryKeyPath = registryKeyPath.replace('USER\\', 'HKEY_CURRENT_USER\\')
+                item["registryKeyPath"] = registryKeyPath
+                if item.get("registryPath") is not None:
+                    registryPath = item.get("registryPath")
+                    registryPath = registryPath.replace('MACHINE\\', 'HKEY_LOCAL_MACHINE\\')
+                    registryPath = registryPath.replace('USER\\', 'HKEY_CURRENT_USER\\')
+                    item["registryPath"] = registryPath
+                if item.get("registryOldValue") is None and item.get("registryValue") is None:
+                    item["registryPath"] = None
+                    item["registryKeyPath"] = None
+                if item.get("registryOldValue") is not None:
+                    oldvalue=item.get("registryOldValue")
+                    valuetype = item.get("registryOldValueType")
+                    if valuetype == "BINARY":
+                        valuetype = "REG_BINARY"
+                        item["registryOldValueType"] = valuetype
+                    registryOldValue = {"data": oldvalue, "data_type": valuetype}
+                    item["registryOldValue"] = registryOldValue
+                if item.get("registryValue") is not None:
+                    rvalue = item.get("registryValue")
+                    valuetype = item.get("registryOldValueType")
+                    if valuetype == "BINARY":
+                        valuetype = "REG_BINARY"
+                        item["registryOldValueType"] = valuetype
+                    registryValue = {"data": rvalue, "data_type": valuetype}
+                    item["registryValue"] = registryValue
+
+            if item.get("loginIsAdministratorEquivalent") is not None:
+                if item.get("loginIsAdministratorEquivalent") == "True" or \
+                        item.get("loginIsAdministratorEquivalent") == "TRUE":
+                    item["loginIsAdministratorEquivalent"] = True
+                elif item.get("loginIsAdministratorEquivalent") == "False" or \
+                        item.get("loginIsAdministratorEquivalent") == "FALSE":
+                    item["loginIsAdministratorEquivalent"] = False
+
+            ResultsConnector.replace_escape_character(item)
+
+            empty_keys = [k for k, v in item.items() if v is None]
+            for key in empty_keys:
+                del item[key]
```

Comment:
you don't need to delete the None value fields at his stage unless you have very specific reason. None value is handled by the json to stix translator.

Reply:
Done the changes.